### PR TITLE
add `bazel`/`bazelisk`

### PR DESCRIPTION
**Pull request by MarkusZoppelt**
add [`bazelisk`](https://github.com/bazelbuild/bazelisk) (`bazel`)

### CODE REVIEW

File: projects/github.com/bazelbuild/bazelisk/package.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+distributable:
+  url: https://github.com/bazelbuild/bazelisk/archive/refs/tags/v{{version}}.tar.gz
+  ref: v{{version}}}
```
Comment (jhheider) [inline projects/github.com/bazelbuild/bazelisk/package.yml]: `ref:` is used with git checkouts; you want `strip-compents: 1`, so the name-version of the package is stripped from the extraction path.

File: projects/github.com/bazelbuild/bazelisk/package.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+versions:
+  github: bazelbuild/bazelisk
+
+build:
+  dependencies:
+    go.dev: '^1.20'
+  script:
+    - go mod download
+    - go build -ldflags "$LDFLAGS" -o "{{prefix}}/bin/bazelisk"
+    - ln -n {{prefix}}/bin/bazelisk {{prefix}}/bin/bazel
```
Comment (jhheider) [inline projects/github.com/bazelbuild/bazelisk/package.yml]: this should definitely be: `ln -s bazelisk {{prefix}}/bin/bazel`. using an absolute path will make it nonportable.

File: projects/github.com/bazelbuild/bazelisk/package.yml (hunk 1 above)
Comment (MarkusZoppelt) [reply projects/github.com/bazelbuild/bazelisk/package.yml]: done!

File: projects/github.com/bazelbuild/bazelisk/package.yml (hunk 2 above)
Comment (MarkusZoppelt) [reply projects/github.com/bazelbuild/bazelisk/package.yml]: changed, thx!

**Comment by jhheider** [comment]
`failed to verify certificate: x509: certificate signed by unknown authority`

needs a runtime-dep on `curl.se/ca-certs`

**Pull request merged**